### PR TITLE
Docs: updates quickstarts with hosted authn service

### PR DESCRIPTION
**Pull request by ZPain8464**
Updates the following:
- Core Docker quickstart
- Enterprise Docker quickstart
- K8s Core quickstart

### CODE REVIEW

File: content/docs/deploying/k8s/quickstart.mdx [hunk 1]
```diff
@@ -23,14 +23,17 @@ import TabItem from '@theme/TabItem';
 import InstallMkcert from '@site/content/_install-mkcert.md';
 import GlobalExample from '@site/content/examples/kubernetes/pomerium-global-settings.md';
 
-This quick-start will show you how to deploy Pomerium with [Kubernetes].
+Deploy Pomerium Core with [Kubernetes].
+
+This quickstart guide uses our **Hosted Authenticate Service** so you don't need to configure an identity provider or authenticate service URL.
+
+If you want to self-host, see the **Self-Hosted Authenticate Service** page.
```
Comment (wasaga) [inline content/docs/deploying/k8s/quickstart.mdx:30]: add links?

File: content/docs/quickstart.mdx [hunk 2]
```diff
@@ -25,74 +25,63 @@ keywords:
...
+# Run Pomerium Core With Docker
 
-## Prerequisites
-
-- A configured [identity provider]
-- [Docker] and [Docker Compose]
+Run Pomerium Core with Docker containers in **under 5 minutes**.
 
-## Configure
+The Core quickstart uses Pomerium's **Hosted Authenticate Service**, but you can also configure a **self-hosted authenticate service** to integrate with Pomerium.
```
Comment (wasaga) [inline content/docs/quickstart.mdx:32]: add links

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 3]
```diff
@@ -63,107 +45,47 @@ This guide assumes you've already registered for Pomerium Enterprise and have cr
...
-  metrics:
+<HACompose />
+
+Next, you need to update your [signing key](/docs/reference/signing-key):
+
+```shell
+# Generates a P-256 (ES256) signing key
+openssl ecparam  -genkey  -name prime256v1  -noout  -out ec_private.pem
+# Prints the base64-encoded value of the signing key
+cat ec_private.pem | base64
```
Comment (wasaga) [inline content/docs/releases/enterprise/install/quickstart.mdx:71]: the signing key is part of docker compose already.

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 4]
```diff
@@ -1,59 +1,41 @@
...
+title: Run Pomerium Enterprise With Docker
 sidebar_label: Quickstart
 description: Demo Pomerium Enterprise
 sidebar_position: 1
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import HAConfig from '../../../../examples/enterprise/hosted-auth-config.yaml.md';
+import HACompose from '../../../../examples/enterprise/hosted-auth-docker.yaml.md';
```
Comment (wasaga) [inline content/docs/releases/enterprise/install/quickstart.mdx:13]: HA is a common abbreviation for High Availability. 

Let's use something like 

```suggestion
import HostedAuthenticateConfig from '../../../../examples/enterprise/hosted-auth-config.yaml.md';
import HostedAuthenticateCompose from '../../../../examples/enterprise/hosted-auth-docker.yaml.md';
```

**Review by wasaga — CHANGES_REQUESTED** [review]

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 5]
```diff
@@ -1,59 +1,41 @@
...
 
-# Pomerium Enterprise Quickstart
+# Run Pomerium Enterprise With Docker
 
-Learn how to run Pomerium Enterprise using Docker Compose.
+Run Pomerium Enterprise with Docker containers and connect to the Console.
+
+This guide uses our Hosted Authenticate Service. If you use our hosted service and have a license key, you can complete this guide in **under 5 minutes**.
```
Comment (desimone) [inline content/docs/releases/enterprise/install/quickstart.mdx:19]:
```suggestion
This guide uses our Hosted Authenticate Service. If you use our hosted service and have a license key, you can complete this guide in **under 5 minutes**.
```

Link to Hosted Authenticate Service capability page.

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 6]
```diff
@@ -1,59 +1,41 @@
...
+
+See the Self-Hosted Authenticate Service page if you want to self-host Pomerium.
```
Comment (desimone) [inline content/docs/releases/enterprise/install/quickstart.mdx:21]: Link :)

File: content/examples/docker/basic.docker-compose.yml.md [hunk 7]
```diff
@@ -1,25 +1,16 @@
 ```yaml
 version: "3"
-networks:
-  main: {}
 services:
   pomerium:
-    image: pomerium/pomerium:latest
+    image: pomerium/pomerium:v0.22.0
```
Comment (desimone) [inline content/examples/docker/basic.docker-compose.yml.md:5]: We should update this to the latest patch _everywhere_. And we need to remember to bump this everytime we do a release or minor.

**Review by desimone — CHANGES_REQUESTED** [review]

File: content/examples/docker/basic.docker-compose.yml.md (hunk 7 above)
Comment (wasaga) [reply content/examples/docker/basic.docker-compose.yml.md:5]: let's just do the `latest => v0.22.0` _everywhere_ part in a separate PR, as I have an incoming PR this one is blocking

File: content/docs/releases/enterprise/install/quickstart.mdx (hunk 6 above)
Comment (ZPain8464) [reply content/docs/releases/enterprise/install/quickstart.mdx:21]: The link isn't available in this PR. I just merged the HA docs in a separate PR. I planned on linking after the HA docs are merged.

File: content/docs/quickstart.mdx (hunk 2 above)
Comment (ZPain8464) [reply content/docs/quickstart.mdx:32]: Links aren't available in this PR. I just merged HA docs, so I'd have to merge into this one to link.

File: content/examples/docker/basic.docker-compose.yml.md (hunk 7 above)
Comment (desimone) [reply content/examples/docker/basic.docker-compose.yml.md:5]: Let's do this in a FUP PR. But let's change the format to use minor'tagged images e.g. (`v0.21`)

**Review by desimone — APPROVED** [review]

File: content/docs/releases/enterprise/install/quickstart.mdx (hunk 3 above)
Comment (wasaga) [reply content/docs/releases/enterprise/install/quickstart.mdx:71]: this step is unnecessary as signing key is already part of docker compose

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 8]
```diff
@@ -63,107 +45,47 @@ This guide assumes you've already registered for Pomerium Enterprise and have cr
...
-  metrics:
+<HostedCompose />
+
+Next, you need to update your [signing key](/docs/reference/signing-key):
+
+```shell
+# Generates a P-256 (ES256) signing key
+openssl ecparam  -genkey  -name prime256v1  -noout  -out ec_private.pem
+# Prints the base64-encoded value of the signing key
+cat ec_private.pem | base64
```
Comment (wasaga) [inline content/docs/releases/enterprise/install/quickstart.mdx:71]:
```suggestion
```

File: content/docs/releases/enterprise/install/quickstart.mdx [hunk 9]
```diff
@@ -63,107 +45,47 @@ This guide assumes you've already registered for Pomerium Enterprise and have cr
...
 ```
 
-Below are a few points to note about services running in `docker-compose`:
+Add the base64-encoded signing key to the `SIGNING_KEY` variables in your `docker-compose.yaml` file. The same signing key should be used in your `pomerium` and `pomerium_console` services.
```
Comment (wasaga) [inline content/docs/releases/enterprise/install/quickstart.mdx:74]: The docker-compose file already has it pre-generated. 

```suggestion
```

**Review by wasaga — CHANGES_REQUESTED** [review]

File: content/examples/docker/basic.docker-compose.yml.md [hunk 10]
```diff
@@ -1,7 +1,5 @@
 ```yaml
 version: "3"
-networks:
-  main: {}
 services:
   pomerium:
     image: pomerium/pomerium:latest
```
Comment (wasaga) [inline content/examples/docker/basic.docker-compose.yml.md:5]:
```suggestion
    image: pomerium/pomerium:v0.22.0
```

File: content/examples/enterprise/hosted-auth-docker.yaml.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
+```yaml title="docker-compose.yaml"
+version: '3'
+services:
+  pomerium:
+    image: pomerium/pomerium:latest
```
Comment (wasaga) [inline content/examples/enterprise/hosted-auth-docker.yaml.md:5]:
```suggestion
    image: pomerium/pomerium:v0.22.0
```

File: content/examples/enterprise/hosted-auth-docker.yaml.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+      - DATABROKER_STORAGE_TYPE=postgres
+      - DATABROKER_STORAGE_CONNECTION_STRING=postgresql://postgres:postgres@database/postgres?sslmode=disable
+      - SIGNING_KEY=LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSUc0R0N4bjlxaDBHRVZnV3VCM0VoRm51RlptZ2VkZXJsMEtLd0ZoRWo4Tk9vQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFMXFOWXNUMFpSZEVTS0djSXRqZFUxcGJZREVDTktRd2lNcmNHVFl6RUhLM1V5MnVoT1N3bgpXVGdWUHppTk4vcWozYXFJeSs3Sk55ZEFLVlo3bURPNGtnPT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=
+  pomerium_console:
+    depends_on:
+      database:
+        condition: service_healthy
+      pomerium:
+        condition: service_started
+    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:latest
```
Comment (wasaga) [inline content/examples/enterprise/hosted-auth-docker.yaml.md:23]:
```suggestion
    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:v0.22.0
```

**Review by wasaga — CHANGES_REQUESTED** [review]
please pin core / console version images.

**Review by wasaga — APPROVED** [review]
ok I see there's FUP PR to deal with version tags https://github.com/pomerium/documentation/pull/586#discussion_r1181718043

Were quickstarts retested @ssveta7ak ?

**Pull request merged**